gatsby-config.js

```diff
@@ -17,9 +17,10 @@ module.exports = {
     'gatsby-transformer-sharp',
     'gatsby-plugin-sharp',
     {
-      resolve: 'gatsby-plugin-google-analytics',
+      resolve: 'gatsby-plugin-google-tagmanager',
       options: {
-        trackingId: process.env.GOOGLE_ANALYTICS_TRACKER_ID,
+        id: 'GTM-TW4FPB3',
```

Comment:
This does not have to be an `ENV`?

Reply:
Yes, I could add it as an environment variable in the CD workflow (when the static pages are built).